automation_tools/satellite6/capsule.py

```diff
@@ -208,11 +208,13 @@ def generate_capsule_certs(capsule_hostname):
 
     :param str capsule_hostname: The fully qualified domain name for the
         capsule.
+    :param bool force: Force creation of the capsule cert even if it is
+    already created.
```

Comment:
@jyejare if you like -  on your robottelo root folder, run 'make docs' and inspect the htmls to see why @elyezer is asking for indentation.